OptimizelySDK/ProjectConfig.cs

```diff
@@ -617,5 +635,15 @@ public string GetAttributeId(string attributeKey)
             Logger.Log(LogLevel.ERROR, $@"Attribute key ""{attributeKey}"" is not in datafile.");
             return null;
         }
+
+        /// <summary>
+        /// Check if the provided experiment Id belongs to any feature, false otherwise.
```

Comment:
We should add a unit test for this method with the `ExperimentFeatureMap` being populated

Reply:
Unit tests added for `IsFeatureExperiment` method.